Nodejs/Product/ProjectWizard/NodejsPackageParametersExtension.cs

```diff
@@ -14,23 +14,36 @@
 //
 //*********************************************************//
 
+using System;
 using System.Collections.Generic;
 using System.Text.RegularExpressions;
 using EnvDTE;
 using Microsoft.VisualStudio.TemplateWizard;
-
+
 namespace Microsoft.NodejsTools.ProjectWizard {
     class NodejsPackageParametersExtension : IWizard {
-        public void RunStarted(object automationObject, Dictionary<string, string> replacementsDictionary, WizardRunKind runKind, object[] customParams) {
-            var projectName = replacementsDictionary["$projectname$"];
+        private const int NpmPackageNameMaxLength = 214;
 
-            // Remove all leading url-invalid, underscore, and period characters from the string
+        /// <summary>
+        /// Normalize a project name to be a valid Npm package name.

```

Comment:
Could you include a link to the docs on valid package names in the comment here?
https://docs.npmjs.com/files/package.json#name